plugins/otter-pro/inc/plugins/class-form-pro-features.php

```diff
@@ -347,6 +348,135 @@ public function send_autoresponder( $form_data ) {
 		}
 	}
 
+	/**
+	 * Send autoresponder email to the subscriber.
+	 *
+	 * @param Form_Data_Request|null $form_data The files to load.
+	 * @since 2.4
+	 */
+	public function trigger_webhook( $form_data ) {
```

Comment:
Just a reminder that we need to add a filter for the data being sent to webhook, with the webhook & form id being a parameter